src/frontend/src/lib/components/transactions/Transaction.svelte

```diff
@@ -0,0 +1,45 @@
+<script lang="ts">
+	import { nonNullish } from '@dfinity/utils';
+	import { BigNumber } from '@ethersproject/bignumber';
+	import type { ComponentType } from 'svelte';
+	import IconReceive from '$lib/components/icons/IconReceive.svelte';
+	import IconSend from '$lib/components/icons/IconSend.svelte';
+	import TransactionPending from '$lib/components/transactions/TransactionPending.svelte';
+	import Amount from '$lib/components/ui/Amount.svelte';
+	import Card from '$lib/components/ui/Card.svelte';
+	import RoundedIcon from '$lib/components/ui/RoundedIcon.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import type { TransactionType } from '$lib/types/transaction';
+	import { formatSecondsToDate } from '$lib/utils/format.utils.js';
+
+	export let value: BigNumber | undefined;
+	export let type: TransactionType;
+	export let pending: boolean;
+	export let timestamp: number | undefined;
+
+	let label: string;
+	$: label = type === 'send' ? $i18n.send.text.send : $i18n.receive.text.receive;
+
+	let icon: ComponentType;
+	$: icon = type === 'send' ? IconSend : IconReceive;
+</script>
+
+<button class="contents">
+	<Card>
+		{label}
+
+		<RoundedIcon slot="icon" {icon} iconStyleClass={pending ? 'opacity-10' : ''} />
+
+		{#if nonNullish(value)}
```

Comment:
This works? Isn't Svelte complaining? Should you use a `<svelte:fragment slot="amount">` and have the `if` finding place within the fragment?

Reply:
Good catch, it wasn't displaying properly. Fixed it.